### ISSUE TITLE
Invalid PEM formatted message

### ISSUE THREAD

**Issue by brenzi** [w3f/polkadot-validator-setup#146]
I have exported env variables
```
TF_VAR_do_token
GOOGLE_APPLICATION_CREDENTIALS
SSH_ID_RSA_PUBLIC
SSH_ID_RSA_VALIDATOR
```
and the ssh keys were generated with empty passphrase using `ssh-keygen -m PEM -f <path>` and added to ssh-agent
(I wasn't able to use keys protected by passphrase, caused another error...)

```
yarn run v1.15.2
$ node . sync -c config/main.json
Syncing platform...

Initializing the backend...


Initializing provider plugins...

- Finding hashicorp/google versions matching "~> 2.15"...

- Installing hashicorp/google v2.20.3...

- Installed hashicorp/google v2.20.3 (signed by HashiCorp)


Terraform has created a lock file .terraform.lock.hcl to record the provider
selections it made above. Include this file in your version control repository
so that Terraform can guarantee to make the same selections by default when
you run "terraform init" in the future.



╷
│ Warning: Version constraints inside provider configuration blocks are deprecated
│
│   on main.tf line 3, in provider "google":
│    3:   version     = "~>2.15"
│
│ Terraform 0.13 and earlier allowed provider version constraints inside the
│ provider configuration block, but that is now deprecated and will be
│ removed in a future version of Terraform. To silence this warning, move the
│ provider version constraint into the required_providers block.
╵

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.


Terraform used the selected providers to generate the following execution
plan. Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # google_storage_bucket.imagestore will be created
  + resource "google_storage_bucket" "imagestore" {
      + bucket_policy_only = (known after apply)
      + force_destroy      = true
      + id                 = (known after apply)
      + location           = "US"
      + name               = "integritee-sv-tf-state"
      + project            = (known after apply)
      + self_link          = (known after apply)
      + storage_class      = "STANDARD"
      + url                = (known after apply)
    }

Plan: 1 to add, 0 to change, 0 to destroy.

google_storage_bucket.imagestore: Creating...

╷
│ Warning: Version constraints inside provider configuration blocks are deprecated
│
│   on main.tf line 3, in provider "google":
│    3:   version     = "~>2.15"
│
│ Terraform 0.13 and earlier allowed provider version constraints inside the
│ provider configuration block, but that is now deprecated and will be
│ removed in a future version of Terraform. To silence this warning, move the
│ provider version constraint into the required_providers block.
╵

╷
│ Error: googleapi: Error 409: You already own this bucket. Please select another name., conflict
│
│   with google_storage_bucket.imagestore,
│   on main.tf line 6, in resource "google_storage_bucket" "imagestore":
│    6: resource "google_storage_bucket" "imagestore" {
│
╵

Command execution failed with code: 1
Allowed error creating state backend: 1
Could not sync platform: Invalid PEM formatted message.
error Command failed with exit code 255.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by brenzi** [comment]
oops, one of the `SSH...` env pointed to the `.pub` instead of private key. never mind
